scripts/tests/testIntegration.sh

```diff
@@ -3,5 +3,9 @@
 # set 'test' environment
 export NODE_ENV='test'
 
+# get project_id value from firestore config
+json=$(node -e "console.log(require('config').get('firestore'))")
+project_id=$(echo $json | grep -o '"project_id":[^,}]*' | cut -d':' -f2 | tr -d '"' | tr -d '[:space:]')
```

Comment:
Brief explanation of the above line

`grep -o '"project_id":[^,}]*`: This command uses grep to search for the pattern "project_id": followed by any number of characters that are not , or }. The -o flag tells grep to only output the matched text.

`cut -d':' -f2`: This command uses cut to extract the second field from the output of grep, using : as the delimiter. Since the output of grep is "project_id": "real-dev-squad-backend-abda3", the second field is "real-dev-squad-backend-abda3".

`tr -d '"'`: This command uses tr to delete any occurrences of " characters from the output of cut.

`tr -d '[:space:]'`: This command uses tr to delete any whitespace characters from the output of the previous command.